src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -1382,6 +1370,80 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
+    Partial Class BoundFieldInitializer
+        Implements IFieldInitializer
+
+        Private ReadOnly Property IInitializedField As IFieldSymbol Implements IFieldInitializer.InitializedField
+            Get
+                Return Me.InitializedFields.FirstOrDefault()
+            End Get
+        End Property
+
+        Private ReadOnly Property IValue As IExpression Implements ISymbolInitializer.Value
+            Get
+                Return Me.InitialValue
+            End Get
+        End Property
+
+        Protected Overrides Function StatementKind() As OperationKind
+            Return OperationKind.FieldInitializerAtDeclaration
+        End Function
+    End Class
+
+    Partial Class BoundPropertyInitializer
+        Implements IPropertyInitializer
+
+        Private ReadOnly Property IInitializedProperty As IPropertySymbol Implements IPropertyInitializer.InitializedProperty
+            Get
+                Return Me.InitializedProperties.FirstOrDefault()
```

Comment:
Within the VB compiler, there is logic for handling multiple properties that share an initializer in addition to multiple fields that share an initializer. Is it actually possible to write VB code in which two properties share an initializer?